esp32-c3-lcdkit/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+idf_component_register(
+    SRCS "esp32-c3-lcdkit.c"
+    INCLUDE_DIRS "include"
+    PRIV_INCLUDE_DIRS "priv_include"
+    PRIV_REQUIRES
+        "esp_lcd"
+        "esp_lcd_touch"
```

Comment:
Please remove `bsp_display_lcd_init` and `esp_lcd_touch` from here. It doesn't have to be here, when you add it into `idf_component.yml`.